lib/chef/secret_fetcher/azure_key_vault.rb

```diff
@@ -8,23 +8,30 @@ class SecretFetcher
     # In this initial iteration this authenticates via token obtained from the OAuth2  /token
     # endpoint.
     #
-    # Usage Example:
+    # Validation of required configuration (vault name) is not performed until
+    # `fetch` time, to allow for embedding the vault name in with the secret
+    # name, such as "myvault/secretkey1".
```

Comment:
if you make this my_vault then spellcheck won't care

Reply:
Can we teach spellcheck to not care about words containing slashes? 

I guess that wouldn't work because eether/or.